### ISSUE TITLE
SVGO options are overridden

### ISSUE THREAD

**Issue by carlcs** [freshheads/svg-spritemap-webpack-plugin#9]
It seems that my SVGO options are being overridden because of how `cleanupIDs: false` plugin is merged into the options.

```js
new SVGSpritemapPlugin({
    src: './src/svg/icons/**/*.svg',
    filename: 'svg/icons-sprite.svg',
    svgo: {
        plugins: [{
            convertColors: {currentColor: '#030303'}
        }],
    },
}),
```

**Comment by cascornelissen** [comment]
That does look like a bug, yeah. Would you be willing to submit a PR for this?

**Comment by carlcs** [comment]
@cascornelissen please have a look at #11

**Comment by cascornelissen** [comment]
This has been fixed in #11 which was included in version `1.0.3` which was just released.

@carlcs, thanks for the help!
